src/main/java/frc/robot/commands/MoveToHeight.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class MoveToHeight extends CommandBase {
+  Elevator m_elevator; 
+  double m_elevatorSetpoint; 
+
+  public MoveToHeight(Elevator elevator, double elevatorHeight) {
+    m_elevator = elevator; 
+    m_elevatorSetpoint = elevatorHeight; 
+    addRequirements(m_elevator);
+  }
+
+  @Override
+  public void initialize(){
+    //set the elevator PID speed to be clamped within the correct range to get it to move faster
+    //This should be done already in ResetEncoderAtBottom() but just in case something happens it's here too. 
+    m_elevator.setMotorLimit();
+
+    m_elevator.setSepointMeters(m_elevatorSetpoint);
+
+  }
+
+  @Override
+  public void execute() {
+    System.out.println(m_elevatorSetpoint-m_elevator.getElevatorHeightMeters());
+    m_elevator.setSepointMeters(m_elevatorSetpoint); 
```

Comment:
The setpoint can't change, so don't set it again. You don't need to do anything in periodic(). If you would like, you could log the error. LogManager.addDouble("Elevator/error", () -> m_pid.getSetpoint - getElevatorHeight()); would work in Elevator.java in the constructor to log the error.

Reply:
I will figure this out a little later.